app/assets/stylesheets/narrow-table-display.scss

```diff
@@ -1,9 +1,5 @@
 // stylelint-disable selector-no-qualifying-type
 
-th.table-combine_right_if_narrow .table-hidden_right_title {
-  display: none;
-}
-
 .table-merge_columns tr {
```

Comment:
Do we still need the remaining styles, would the GOVUK defaults suffice?
If required, could we consolidate all table styles within `app/assets/stylesheets/tables.scss`